### PR TITLE
Fix lint issues

### PR DESCRIPTION
**Pull request by fjvallarino**
@santiagonoguez @olegrise please review

### CODE REVIEW

File: web/scripts/template-editor/directives/dtv-basic-storage-selector.js [hunk 1]
```diff
@@ -60,37 +62,42 @@ angular.module('risevision.template-editor.directives')
...
-              console.log('Failed to load files', err);
-            })
-            .finally(function () {
-              $loading.stop(spinnerId);
-            });
           };
 
           $scope.selectItem = function (item) {
-            templateEditorUtils.addOrRemove($scope.selectedItems, { name: item.name }, item);
+            templateEditorUtils.addOrRemove($scope.selectedItems, {
```
Comment (andrecardoso) [inline web/scripts/template-editor/directives/dtv-basic-storage-selector.js]: I think the lint friendly version is worse for reading. Can we tweak the rules to allow this?

**Comment by olegrise** [comment]
@fjvallarino if I understand it correctly, the changes in this PR is a result of using [jsbeautifier](https://github.com/Rise-Vision/rise-vision-apps/blob/master/gulpfile.js#L142), not lint.
 
@andrecardoso I don't mind if we tweak the rules, but I'm also okay with the current rules.

File: web/scripts/template-editor/directives/dtv-basic-storage-selector.js (hunk 1 above)
Comment (fjvallarino) [reply web/scripts/template-editor/directives/dtv-basic-storage-selector.js]: @andrecardoso I totally agree! There are a bunch of things the beautifier does that makes the situation worse. Maybe we should all discuss what we want our code to look like?

**Comment by fjvallarino** [comment]
@olegrise yeah, it's the beautifier, not the linter. I'm going to close this PR until we reach an agreement on how we want the code to look like

@andrecardoso @santiagonoguez FYI

**Comment by olegrise** [comment]
@fjvallarino up to you, but I would merge this PR and then work on reaching an agreement. You've got 3 approvals 😉

**Pull request closed without merging**